src/server/mods.cpp

```diff
@@ -66,8 +66,14 @@ void ServerModManager::loadMods(ServerScripting *script)
 		}
 		std::string script_path = mod.path + DIR_DELIM + "init.lua";
 		infostream << "  [" << padStringRight(mod.name, 12) << "] [\""
-			   << script_path << "\"]" << std::endl;
+			<< script_path << "\"]" << std::endl;
+		using namespace std::chrono;
```

Comment:
don't do this, it's a little bit ugly inside the code. you can use the full namespace for three calls it's not problematic

Reply:
ok